### ISSUE TITLE
Occassional github-pages failures

### ISSUE THREAD

**Issue by adowling2** [ndcbe/optimization#2]
@jckantor Have you seen this before?

https://github.com/ndcbe/optimization/actions/runs/3961768772

**Comment by jckantor** [comment]
Yes, I’ve seen that on occasion, and not sure of the origin. It seems connected with GitHub actions, though I don’t often use actions in my repos.  Wish I had better information to offer.

**Comment by adowling2** [comment]
I confirmed there was a GitHub pages outage around this time. Let's see if it happens again.

**Comment by jckantor** [comment]
Good to know. We do get this from time to time.
